src/theory/arith/constraint.cpp

```diff
@@ -1196,6 +1201,15 @@ void Constraint::impliedByIntHole(ConstraintCP a, bool nowInConflict){
 }
 
 void Constraint::impliedByIntHole(const ConstraintCPVec& b, bool nowInConflict){
+  Debug("constraints::pf") << "impliedByIntHole(" << this;
+  if (Debug.isOn("constraints::pf")) {
+    for (const ConstraintCP p : b)
```

Comment:
Nit: `const ConstraintCP&`

Reply:
Question: Do you always have your iteratee be a reference, even if the type is small enough for copying to be a non-issue?

Reply:
Not for base types like int, bool, ... ConstraintCP is a pointer, right? Didn't realize that this is the case when I reviewed, so in this case not using a reference is fine. Doesn't hurt, though.